File: Programmers/Lv0/배열만들기3.py
def solution(arr, intervals):
    answer = []
    for i in range(intervals[0][0], intervals[0][1] + 1):
        answer.append(arr[i])
    for j in range(intervals[1][0], intervals[1][1] + 1):
        answer.append(arr[j])
    return answer

File: Programmers/Lv0/배열만들기3.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(vector<int> arr, vector<vector<int>> intervals) {
    vector<int> answer;
    for (int idx = intervals[0][0]; idx <= intervals[0][1]; idx++)
        answer.push_back(arr[idx]);
    for (int idx = intervals[1][0]; idx <= intervals[1][1]; idx++)
        answer.push_back(arr[idx]);
    return answer;
}